test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -216,6 +226,8 @@ public void transactionsSpanningReconfigurationTest() throws Exception {
         }
 
         assertThat(txnFailed).isTrue();
+
+        runtime.shutdown();
```

Comment:
Should runtime be set to null to prevent double shutdown during shutdownRuntime() ?

Reply:
Good catch! will move to AbstractIT and address it.

Reply:
Done, set to null in setUp() method which will be called before any test.